modules/Link.js

```diff
@@ -22,6 +22,10 @@ function isEmptyObject(object) {
   return true
 }
 
+function resolveToLocation(to, router) {
```

Comment:
I would make this a class method so you don't have to pass it any args Bonus points: You can make it a getter (`router.push(this.to)`). 

Also, and this is personal preference, but I'm generally not a fan of extracting methods from a class unless they are also being exported from the module.